### ISSUE TITLE
it's terrible that doc not including the instructions about installing

### ISSUE THREAD

**Issue by jackwener** [charleskorn/kaml#22]
how annotate @Serializable
how data class own serializer mentod

**Comment by charleskorn** [comment]
Hi @jackwener, take a look at the documentation for kotlinx.serialization - https://github.com/Kotlin/kotlinx.serialization/. It explains how to use the things you mention.

**Comment by jackwener** [comment]
> Hi @jackwener, take a look at the documentation for kotlinx.serialization - https://github.com/Kotlin/kotlinx.serialization/. It explains how to use the things you mention.

Yes, I have notice this. I think that it would be better if there is explanation in the doc
